docs/blog/newsletter-july-2019.rst

```diff
@@ -0,0 +1,115 @@
+.. post:: July 3, 2019
+   :tags: newsletter
+
+#####################################
+Write the Docs Newsletter – July 2019
+#####################################
+
+Hi, WTD friends, and welcome to the July newsletter! It's Beth A here writing from sweltering Europe, where I've been learning great new words like *canicule* and *Hitzewelle* - aka heatwave in French and German.
+
+(Fun fact: *canicule* comes from the Latin *dies caniculares*, or "dog days" - the most uncomfortable part of summer, so named because they come after the `helical rising <https://en.wikipedia.org/wiki/Heliacal_rising>`_ of Sirius, the Dog Star. Well, I thought it was interesting.)
+
+Aside from fascinating facts about the weather, we do have some actual news. Firstly, `tickets are on sale for the Australia conference </conf/australia/2019/news/au-2019-welcome/>`__, and `the call for proposals is also open </conf/australia/2019/cfp/>`__! The CFP closes on 9 August, and the conference itself is 14-15 November in Melbourne.
+
+Here in Europe, you'll be pleased to know that `Prague speakers have been announced </conf/prague/2019/news/talks-volunteers-tickets-shirts/>`__! `Here's the full list </conf/prague/2019/speakers/>`__, and you can grab your tickets `here </conf/prague/2019/tickets/>`__. And if you're interested in helping out behind the scenes, we'd love you to `volunteer at the conference </conf/prague/2019/news/talks-volunteers-tickets-shirts/#call-for-volunteers>`__.
+
+.. TODO announce interview stuff, if it's published in time
+
+So that's the news. What have we been talking about on Slack this month?
+
+-----------------------------------
+Making the leap to managing writers
+-----------------------------------
+
+One of our big discussions this month was about folks trying to weigh up opportunities to move into managing writers. Here's the guidance the community gave.
+
+There are definitely some things to be wary of. If writing's your passion, moving to management means much less time for it. This can mean losing touch with the craft, and leaving your portfolio lighter if you decide to back out. Another issue is that you're accountable for both the successes and failures of your team, which can be stressful. 
+
+But the upsides of management are the bigger projects that can challenge you in different ways, including helping your team reach their full potential. In the long run, teaching others lets you achieve than you would have been able to on your own.
+
+If you'll be hiring, recruitment can create a more direct sense of accomplishment. On the flip side, you may also at some point be responsible for letting people go, a daunting task.
```

Comment:
Yeah, previous paragraphs could imitate the rhetorical structure of this one.

Reply:
I think I tried to do that with the previous two paragraphs, just split across two paras? Anyway, I'm just gonna leave it for now, as I've done a lot of tweaking.